docs/docs/dev/deployments/multi-zone.md

```diff
@@ -419,32 +497,39 @@ You can think of this failure case as *"Freezing"* the zone mesh configuration.
 Communication will still work but changes will not be reflected on existing data plane proxies.
 :::
 
-### Communication between Global and Zone control-plane failing
+### Communication between Global and Zone control plane failing
 
-This can happen with mis-configuration or network connectivity issues between control-planes.
+This can happen with misconfiguration or network connectivity issues between control planes.
 
-* Operations inside the zone will happen correctly (dataplane proxies can join and leave and all configuration will be updated and sent correctly).
-* Policy changes will not be propagated to the zone control-plane.
-* Zone ingress and dataplane changes will not be propagated to the global control-plane.
+* Operations inside the zone will happen correctly (data plane proxies can join, leave and all configuration will be updated and sent correctly).
+* Policy changes will not be propagated to the zone control plane.
+* Zone ingress, zone egress and data plane changes will not be propagated to the global control plane:
```

Comment:
I could imagine:
```suggestion
* `ZoneIngress`, `ZoneEgress` and `Dataplane` changes will not be propagated to the global control plane:
```
because it's the resources that are changing, or:
```suggestion
* Zone ingress, zone egress and data plane proxy changes will not be propagated to the global control plane:
```